### PR TITLE
Docs to ES6: Displaying data & Multiple Components pages (#3710)

### PR DESCRIPTION
**Pull request by gillchristian**

### CODE REVIEW

File: docs/docs/02-displaying-data.md [hunk 1]
```diff
@@ -36,16 +36,12 @@ Let's look at a really simple example. Create a `hello-react.html` file with the
...
-  render: function() {
-    return (
-      <p>
-        Hello, <input type="text" placeholder="Your name here" />!
-        It is {this.props.date.toTimeString()}
-      </p>
-    );
-  }
-});
+const HelloWorld = ({date}) => (
```
Comment (jimfb) [inline docs/docs/02-displaying-data.md]: @sebmarkbage What was the conclusion / final opinion here?  I remember there being a poll, and IIRC, you voted against this pattern.

**Comment by facebook-github-bot-9** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

File: docs/docs/02-displaying-data.md (hunk 1 above)
Comment (jimfb) [inline docs/docs/02-displaying-data.md]: Specifically, I'm referring to doing inline destructuring of the arguments.  But I suppose that was in the context of default props.

**Comment by facebook-github-bot-7** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

**Comment by gaearon** [comment]
Thank you for contributing!

I have tweaked this to be a little more conservative so that people new to ES5 don’t get confused when possible. We are working on a docs revamp right now so things might get a little turbulent!

I have merged it as 7f4564a9e73c2a64534fd3f954216afc9a0e5146 (the commit is still attributed to you!)

**Pull request closed without merging**